File: src/main/python/myClient.py
import os, sys

dir = os.path.dirname(__file__)
if not dir:
    sys.path.append('./gen-py')
else:
    sys.path.append('%s/%s' % (dir, '/gen-py'))

from thrift import Thrift
from thrift.transport import TSocket
from thrift.transport import TTransport
from thrift.protocol import TBinaryProtocol
from storm import DistributedRPC

socket    = TSocket.TSocket('localhost', 3772)
transport = TTransport.TFramedTransport(socket)
protocol  = TBinaryProtocol.TBinaryProtocol(transport)
client = DistributedRPC.Client(protocol)

try:
    transport.open()

    print client.execute("dump", "")
    print client.execute("pathRequests", "")
    print client.execute("user-agents", "")
    print client.execute("filtered-user-agents", "Chrome")

except Thrift.TException, tx:
    print "%s" % (tx.message)
finally:
    transport.close()

File: src/main/ruby/myClient.rb
$LOAD_PATH.unshift File.join(File.dirname(__FILE__), 'gen-rb')

require 'rubygems'
require 'thrift'
require 'distributed_r_p_c'

begin
  socket    = Thrift::Socket.new('localhost', 3772)
  transport = Thrift::FramedTransport.new(socket)
  protocol  = Thrift::BinaryProtocol.new(transport)
  client    = DistributedRPC::Client.new(protocol)

  transport.open

  puts client.execute("dump", "").inspect
  puts client.execute("pathRequests", "").inspect
  puts client.execute("user-agents", "").inspect
  puts client.execute("filtered-user-agents", "Chrome").inspect

  transport.close

rescue Thrift::Exception => tx
  print 'Thrift::Exception: ', tx.message, "\n"
end